cmd/jujud/agent/caasoperator.go

```diff
@@ -247,7 +247,7 @@ func (op *CaasOperatorAgent) Workers() (worker.Worker, error) {
 		}
 		return nil, err
 	}
-	if err := startIntrospection(introspectionConfig{
+	if err := StartIntrospection(IntrospectionConfig{
```

Comment:
Anything that you are looking to change to be exported from this package should probably be moved to a dedicated package for that thing, and then used here. Otherwise we are pulling in packages into the dependency tree we don't want.